packages/dm-core-plugins/src/generic-list/GenericListPlugin.tsx

```diff
@@ -25,27 +54,154 @@ export const GenericListPlugin = (
     idReference,
     2
   )
-  if (loading) return <Loading />
+  const [items, setItems] = useState<TGenericObject[]>([])
```

Comment:
is TGenericObject[] type correct for items? 

in the code below it looks like items is a dictionary with a random UID as key + dictionary as value, and not a list.